### PR TITLE
Created XBox Arm controller, Reformatted arm_node, launch files and config files, and improved documentation

### PR DESCRIPTION
**Pull request by GogiPuttar**
Functional X-box arm controller tested on A-2085-05 and A-2085-06 models

[HEBI_XBox.webm](https://github.com/HebiRobotics/hebi_ros2_examples/assets/59332714/cffcf79d-7ddb-4bc2-9b70-6dfb6cb5739b)
[HEBI_XBox.webm](https://github.com/HebiRobotics/hebi_ros2_examples/assets/59332714/cffcf79d-7ddb-4bc2-9b70-6dfb6cb5739b)

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -62,6 +62,24 @@ if(moveit_ros_planning_interface_FOUND)
   )
 endif()
 
+if(moveit_ros_planning_interface_FOUND)
```
Comment (hr1799) [inline CMakeLists.txt:65]: Does the xbox_arm node need moveit? If not, remove the if condition and the dependencies

File: config/A-2084-01_params.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-arm_node:
+/**:
   ros__parameters:
     names: [ "J1_base", "J2_shoulder", "J3_elbow", "J4_wrist" ]
     families: [ "Arm" ]
```
Comment (hr1799) [inline config/A-2084-01_params.yaml:4]: Change families to "HEBIArm" in all config files for consistency.

**Review by hr1799 — CHANGES_REQUESTED** [review]
I have made two comments. Once updated, I can merge the branch.

File: CMakeLists.txt [hunk 3]
```diff
@@ -62,6 +62,21 @@ if(moveit_ros_planning_interface_FOUND)
...
 endif()
 
+add_executable(xbox_arm src/kits/arm/xbox_arm.cpp)
+ament_target_dependencies(xbox_arm
+  rclcpp
+  rclcpp_action
+  trajectory_msgs
+  hebi_msgs
+  geometry_msgs
+  moveit_msgs
```
Comment (hr1799) [inline CMakeLists.txt:72]: Remove moveit dependencies -- line 72-74

File: CMakeLists.txt [hunk 4]
```diff
@@ -142,6 +157,7 @@ install(TARGETS
 if(moveit_ros_planning_interface_FOUND)
   install(TARGETS
     move_arm
+    xbox_arm
```
Comment (hr1799) [inline CMakeLists.txt:160]: move to previous install (without moveit condition dependent)

**Comment by GogiPuttar** [comment]
Made these changes and pushed to prototypes branch. Also included arm_node.launch in xbox_arm.launch

**Review by hr1799 — APPROVED** [review]
Looks good!

**Pull request merged**